tide_core.install

```diff
@@ -212,3 +212,18 @@ function tide_core_update_8005() {
   $config_location = [drupal_get_path('module', 'tide_core') . '/config/optional'];
   _tide_import_single_config('prlp.settings', $config_location);
 }
+
+/**
+ * Enable the blockquote template available to all Node types.
+ */
+function tide_core_update_8006() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_publication') . '/config/install'];
```

Comment:
@vincent-gao  why does the contrib `tide_core` import a config from `tide_publication` which is a `content-vic` only module?

Reply:
thanks sonny.